### PR TITLE
[2.13.X] DDF-4724 documented catalog import and export commands

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?
adds documentation for the `catalog:import` and `catalog:export` commands.

#### Select relevant component teams: 
@codice/docs 

#### Ask 2 committers to review/merge the PR and tag them here.

@brendan-hofmann
@brjeter
@clockard
@lessarderic
@ricklarsen - Documentation
@ryeats
@rzwiefel
@shaundmorris

#### How should this be tested?

review new content
#### What are the relevant tickets?
For GH Issues:
Fixes: #4724 

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by brjeter** [comment]
build now

File: distribution/docs/src/main/resources/content/_running/catalog-commands.adoc [hunk 1]
```diff
@@ -34,6 +34,12 @@ Deprecated as of ddf-catalog 2.5.0. Please use `platform:envlist`.
 
 Provides a list of environment variables.
 
+|catalog:export
+|Exports Metacards and history from the current Catalog
```
Comment (tyler30clemens) [inline distribution/docs/src/main/resources/content/_running/catalog-commands.adoc]: missing period after Catalog

**Comment by tyler30clemens** [comment]
Hero Successful ✅ -- note comment mentioned above though

File: distribution/docs/src/main/resources/content/_running/catalog-commands.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_running/catalog-commands.adoc]: fixed

**Pull request merged**